src/test/app/forms/models/individualDetails.ts

```diff
@@ -6,13 +6,15 @@ import { PartyType } from 'common/partyType'
 import { Address, ValidationErrors as AddressValidationErrors } from 'forms/models/address'
 import { ValidationErrors as CorrespondenceAddressValidationErrors } from 'forms/models/correspondenceAddress'
 import { ValidationError, Validator } from '@hmcts/class-validator'
-import { expectValidationError } from 'test/app/forms/models/validationUtils'
+import { expectValidationError, expectValidationErrorNotPresent } from 'test/app/forms/models/validationUtils'
 import { LocalDate } from 'forms/models/localDate'
+import { ValidationErrors as SplitNamedPartyDetailsValidationErrors } from 'forms/models/splitNamedPartyDetails'
+
 const validAddress = new Address('line1', 'line2', 'line3', 'city', 'bb127nq')
 
 const aVeryLongString = (): string => {
```

Comment:
Would be nice to replace this with `validationUtils.generateString`

Reply:
👍 didn't add that but yeah, if we already have something in place it makes sense to use it

Reply:
Thank you